go/obscuronode/nodecommon/enclave.go

```diff
@@ -0,0 +1,79 @@
+package nodecommon
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/obscuronet/obscuro-playground/go/obscurocommon"
+)
+
+// Enclave - The actual implementation of this interface will call an rpc service
```

Comment:
this comment should be change as well, as it does that now

Reply:
Addressed.